src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -309,6 +317,18 @@ int halide_hexagon_remote_run(handle_t module_ptr, handle_t function,
         printf("Hexagon simulator executed function 0x%x in %d cycles\n", function, cycles);
     }
 
+    if (getenv("HL_HEX_SIM_STATS")) {
+        HEXAPI_Status status = HEX_STAT_SUCCESS;
```

Comment:
Can you declare this on the line below where it's used? HEXAPI_Status status = ...